CMakeLists.txt

```diff
@@ -171,6 +177,26 @@ if (WIN32)
   if (NOT OE_BASH)
     message(FATAL_ERROR "Git Bash not found!")
   endif ()
+
+  # Looking for Perl and Dos2unix if BUILD_OPENSSL is ON.
+  if (BUILD_OPENSSL)
+    # OpenSSL relies on Perl to generate files (e.g., headers, assembly files, and test cases).
+    find_program(OE_PERL perl PATHS "C:/Program Files/Git/bin"
+                                    "${GIT_DIR}/../usr/bin" NO_DEFAULT_PATH)
+    if (NOT OE_PERL)
+      message(FATAL_ERROR "Perl not found!")
+    endif ()
+
+    # Dos2unix utility is used to convert CRLF to LF, which is required by some
+    # OpenSSL tests.
+    find_program(
+      OE_DOS2UNIX dos2unix PATHS "C:/Program Files/Git/bin"
+                                 "${GIT_DIR}/../usr/bin" NO_DEFAULT_PATH)
+    if (NOT OE_DOS2UNIX)
+      message(WARNING "dos2unix not found. Few OpenSSL tests will be disabled.")
```

Comment:
It would be better to make this an ERROR.

Reply:
Done.